lib/rubocop/cop/rspec/each.rb

```diff
@@ -0,0 +1,53 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that all matcher is used instead of iterating over an array.
+      #
+      # @example
+      #   # bad
+      #   it 'validates users' do
+      #     [user1, user2, user3].each { |user| expect(user).to be_valid }
+      #   end
+      #
+      #   # good
+      #   it 'validates users' do
+      #     expect([user1, user2, user3]).to all(be_valid)
+      #   end
+      class Each < Cop
```

Comment:
Yeah, i just think it's definitely possible if not likely. There's already a few cops (for instance the `ItBehavesLike` cop i just added...) that cover lots of potential recommendations (who's to say we won't have more corrections for using `it_behaves_like` in the future) so I think we should start being more careful about how we name new cops.

At the risk of going slightly off topic, there's also https://github.com/backus/rubocop-rspec/issues/232 to consider at some point. Idk if I'm sold on it exactly, but another area where this was slightly awkward recently was adding a new feature to the `ExampleWording` cop. It didn't make sense to add a second wording cop but the two features inside there now would probably be in separate cops and individually configurable, in an ideal world.